test/helpers/minitest_spec_context.rb

```diff
@@ -0,0 +1,9 @@
+require 'minitest/spec'
```

Comment:
we use `support` directory in `test/lib`, we can name this directory `support` as well, for consistency